packages/airnode-node/src/evm/fulfillments/api-calls.ts

```diff
@@ -123,12 +111,8 @@ async function testAndSubmitFulfill(
     return [submitLogs, submitErr, submitData];
   }
 
-  // Compute signature once.
-  // Not computing this beforehand because it might be a very computational expensive operation
-  const signature = await signResponseMessage(request, options.masterHDNode);
-
   // Should not throw
-  const [testLogs, testErr, testData] = await testFulfill(airnodeRrp, { ...request, signature }, options);
+  const [testLogs, testErr, testData] = await testFulfill(airnodeRrp, { ...request }, options);
```

Comment:
Late but I was wondering if we really need to spread the `request` object here or if we could just pass it as is to `testFulfill()`

Reply:
Of course, we can :man_facepalming: I'll add it to my next PR.